### PR TITLE
Adds A *bruh Emote

### PR DESCRIPTION
**Pull request by Dahlular**
## About The Pull Request
Adds a 100% necessary *bruh emote, complete with sound.

## Why It's Good For The Game
Bruh Energy can only be contained for so long. When one picks up their Bruh Momentum to a point of critical mass, it must be released. This is where *bruh comes in, unleashing and defining a Bruh Moment.
In seriousness, it's just a simple addition for the sake of me figuring out github. Please tear into my any mistakes I have made.

## Changelog
:cl:
add: *bruh
/:cl:

### CODE REVIEW

**Comment by Keekenox** [comment]
Bruh

**Comment by Keekenox** [comment]
SMH if this doesn't get added then I have lost all faith in citadel.

**Comment by ghost** [comment]
> SMH if this doesn't get added then I have lost all faith in citadel.

it'll either get testmerged and then not merged or not merged at all
which will be a **bruh** moment

File: modular_citadel/code/modules/mob/cit_emotes.dm [hunk 1]
```diff
@@ -213,3 +213,20 @@
...
+	emote_type = EMOTE_AUDIBLE
+	mob_type_allowed_typecache = list(/mob/living/carbon/)
+
+/datum/emote/living/bruh/run_emote(mob/living/user, params)
+	if(ishuman(user))
+		if(user.nextsoundemote >= world.time)
+			return
+		user.nextsoundemote = world.time + 7
+		playsound(user, 'modular_citadel/sound/voice/bruh.ogg', 50, 1, -1)
+	. = ..()
```
Comment (ghost) [inline modular_citadel/code/modules/mob/cit_emotes.dm:232]: bruh

**Comment by Dahlular** [comment]
> > SMH if this doesn't get added then I have lost all faith in citadel.
> 
> it'll either get testmerged and then not merged or not merged at all
> which will be a **bruh** moment

Real bruh energy right there pardner

File: modular_citadel/code/modules/mob/cit_emotes.dm (hunk 1 above)
Comment (Dahlular) [reply modular_citadel/code/modules/mob/cit_emotes.dm:232]: bruh

**Comment by deathride58** [comment]
ＢＲＵＨ

**Comment by silicons** [comment]
sorry too memey

**Comment by ghost** [comment]
bruh

**Comment by Dahlular** [comment]
> sorry too memey

F

**Pull request closed without merging**